.github/zizmor.yml

```
@@ -0,0 +1,7 @@
+# Zizmor config
+rules:
+  unpinned-uses:
+    config:
+      policies:
+        # Allow pin by ref/tag
+        actions-rust-lang/setup-rust-toolchain: ref-pin
```

Since we're redefining the policy for `unpinned-uses`, we may need to reintroduce zizmor's default ref-pin policy for official GitHub actions.
```suggestion
        actions-rust-lang/setup-rust-toolchain: ref-pin
        actions/*: ref-pin
```

---

Alright, I'll handle this on #2055.